src/main/java/seedu/address/logic/parser/contact/CListCommandParser.java

```diff
@@ -26,27 +29,30 @@ public CListCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
             ArgumentTokenizer.tokenize(args, PREFIX_PHONE, PREFIX_EMAIL,
                 PREFIX_TELEGRAM, PREFIX_ADDRESS, PREFIX_ZOOM, PREFIX_TAG);
```

Comment:
Small issue but wrapped lines should have 8 spaces according to Java coding standard.